src/temporary_home/ResumeCache.h

```diff
@@ -95,6 +104,8 @@ class ResumeCache {
 
   std::unordered_map<StreamId, ResumePosition> streamMap_;
 
+  std::unordered_map<StreamId, std::string /* streamName */> streamNameMap_;
```

Comment:
I think I would prefer this being something like `pendingStreams_` that where the name only exists until the first frame is send (at that point it's added to the streamMap). Thoughts?